_source/api/logzio-public-api.yml

```diff
@@ -177,6 +177,9 @@ tags:
 
 
       **Note:** Shared token endpoints require permissions that must be set by our Support team. Please email help@logz.io for assistance.
+  - name: Manage API tokens
+    description: >-
+      You can manage API tokens for main and sub accounts.
```

Comment:
why main?

Reply:
What would you suggest? 
Because this is the generic parent topic for all API tokens, seemed logical to include main account tokens.

Reply:
Currently you cannot manage tokens of main account. So i'd remove "main"

Reply:
Done - (currently line 182 of my CSV)

Reply:
DONE